contracts/AedexV2Factory.aes

```diff
@@ -42,7 +42,9 @@ contract AedexV2Factory =
 
     entrypoint all_pairs_length(): int = List.length(state.all_pairs)
 
-    entrypoint all_pairs(ix: int): IAedexV2Pair = List.get(ix, state.all_pairs)
+    entrypoint all_pairs(): list(IAedexV2Pair) = state.all_pairs
+
+    entrypoint get_nth_pair(ix: int): IAedexV2Pair = List.get(ix, state.all_pairs)
```

Comment:
`get_nth_pair` seems weird, why not `get_all_pairs()` and `get_pair(idx)`? I think people should be able to read function signatures. Other opinions?

Reply:
`get_pair` and `get_all_pairs` sounds good to me.

Reply:
`get_pair` already exists: gets the pair address based on the given token addresses 
https://github.com/aeternity/dex-contracts-v2/blob/5c0b0296f38d4b974cace6b2cf10e8135e505f94/contracts/AedexV2Factory.aes#L55-L58

Reply:
get_nth_pair is fine by me in this case